symlinkproctor.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+mkdir  ~/.gradle/local-repo/maven/com
+mkdir  ~/.gradle/local-repo/maven/com/indeed
```

Comment:
the above two lines can be replaced with `mkdir -p ~/.gradle/local-repo/maven/com/indeed`